### DISCUSSION TITLE
Support for React Navigation

### DISCUSSION THREAD

**Discussion by visibly** [lodev09/react-native-true-sheet#26] [General]
First of all - great work! Perfect performance and animations.

Do you have any plans for integration of React Navigation (Native Stack Navigator) support _inside_ the sheet?

**Comment by lodev09** [comment]
Yeah I was thinking about it but didn't get a chance to really dig deep into it.

I'll consider it in the future when I get a chance. Or if someone is willing to implement it, I'm open for suggestions.

Thanks for your feedback ☺️

**Comment by lodev09** [comment]
This is now possible! related PR: https://github.com/lodev09/react-native-true-sheet/pull/213
Updated troubleshooting: https://sheet.lodev09.com/troubleshooting#navigating-from-sheet
